sdk/core/azure-core/src/samples/java/com/azure/core/util/ContextJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Code snippets for {@link Context}
+ */
+public class ContextJavaDocCodeSnippets {
+
+    /**
+     * Code snippet for {@link Context#Context(Object, Object)}
+     */
+    public void constructContextObject() {
+        // BEGIN: com.azure.core.util.context#object-object
+        final String key1 = "Key1";
+        final String value1 = "first-value";
+
+        // Create an empty context having no data
+        Context emptyContext = Context.NONE;
+
+        // Create a context using the provided key value pair
+        Context keyValueContext = new Context(key1, value1);
+        // END: com.azure.core.util.context#object-object
+    }
+
+    /**
+     * Code snippet for creating Context object using key-value pair map
+     */
+    public void contextOfObject() {
+        // BEGIN: com.azure.core.util.context.of#map
+        final String key1 = "Key1";
+        final String value1 = "first-value";
+        Map<Object, Object> keyValueMap = new HashMap<>();
+        keyValueMap.put(key1, value1);
+
+        // Create a context using the provided key value pair map
+        Context keyValueContext = Context.of(keyValueMap);
+        System.out.printf("Key1 value %s%n", keyValueContext.getData(key1).get().toString());
```

Comment:
The .toString() isn't necessary, the `%s` formatter will call .toString() on it.